carrent/carrentapp/templates/carrentapp/base.html

```diff
@@ -0,0 +1,51 @@
+{% load static %}
+<! DOCTYPE html>
+<html lang="pl">
+<head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
+    <meta charset="UTF-8">
+    <title> {% block sitle_title %} Car rent {% endblock %}</title>
+    <link href="{% static 'carrentapp/styles/layout.css' %}" rel="stylesheet" type="text/css">
+</head>
+<body>
+<div id="header">
+    <nav class="navbar sticky-top navbar-light bg-#aaaaaa">
+      <div class="container-sm">
+        <a class="navbar-brand" href="#">Strona główna</a>
+        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseExample" role="button" aria-expanded="false" aria-controls="collapseExample">
+            Oferta</a>
+        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseExample" role="button" aria-expanded="false" aria-controls="collapseExample">
+            Wynajem</a>
+        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseExample" role="button" aria-expanded="false" aria-controls="collapseExample">
+            Flota</a>
+        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseExample" role="button" aria-expanded="false" aria-controls="collapseExample">
+            O nas</a>
+        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseExample" role="button" aria-expanded="false" aria-controls="collapseExample">
+            Kontakt</a>
+        <a href="{% url 'login' %}" class="btn btn-primary">Zaloguj</a>
```

Comment:
Poleciłbym dodać funkcje 
{% if request.user.is_authenticated %}
Wtedy jak ktoś się zaloguje to zamiast ciągłego "ZALOGUJ" będzie przycisk WYLOGUJ